src/components/modal/Modal.jsx

```diff
@@ -55,8 +55,15 @@ export const EditSearchModal = ({
   color,
   onChangeColor,
 }) => {
+
+  const handleKeyPress = (event) => {
+    if (event.key === 'Enter') {
+      editItem();
+    }
```

Comment:
should add an if else for Esc key. in which case, the modal should be cancelled. Also should add this function for the other modals (add custom and add search)